src/database/models/sessionOwnerships.js

```diff
@@ -17,6 +17,14 @@ module.exports = (sequelize, DataTypes) => {
 				allowNull: false,
 				primaryKey: true,
 			},
+			type: {
+				type: DataTypes.STRING,
+				allowNull: false,
+				primaryKey: true,
+				validate: {
+					isIn: [['MENTOR', 'CREATOR']],
```

Comment:
Please post the validation error that happens when there is a validation and the response the server would give.

Reply:
@joffinjoy ValidationError [SequelizeValidationError]: Validation error: Validation isIn on type failed

Reply:
@joffinjoy added specific error handling in sessionOwnership creation function.